styles.css

```diff
@@ -35,33 +35,30 @@ body {
   width: 84%;
 }
 #body-modelS {
-  background: url("C://tesla-website/images/modelS.jpg");
+  background: url("./images/modelS.jpg");
   background-repeat: no-repeat;
   background-size: 100% auto;
```

Comment:
Can you add please this line so you cover the image for the entire container?
`background-size: cover;`